### ISSUE TITLE
Bibliography component internal server error

### ISSUE THREAD

**Issue by andrewhercules** [opentargets/platform#1648]
On the [pericarditis (EFO_0007427) profile page](https://platform.opentargets.org/disease/EFO_0007427), the Bibliography widget shows an error.

Can we please investigate and fix?

![Screenshot 2021-07-20 at 17 37 29](https://user-images.githubusercontent.com/7490258/126362446-f9336e5e-03b7-4706-acce-cae18cec6f3f.png)

**Comment by andrewhercules** [comment]
Please note that error is also seen on the [myocarditis (EFO_0009609) profile page](https://platform.opentargets.org/disease/EFO_0009609)

**Comment by d0choa** [comment]
The problem seems to be related with the BE sometimes returning diseases as a `null` object. For example:

Query

```
query SimilarEntitiesQuery{
  disease(efoId: "EFO_0009609") {
    id
    name
    similarEntities(threshold: 0.5, size: 4, entityNames: ["disease"]) {
      score
      object {
        ... on Disease {
          id
          name
        }
      }
    }
  } 
}
```

Response:

```
{
  "data": {
    "disease": {
      "id": "EFO_0009609",
      "name": "myocarditis",
      "similarEntities": [
        {
          "score": 1,
          "object": {
            "id": "EFO_0009609",
            "name": "myocarditis"
          }
        },
        {
          "score": 0.7303702500051845,
          "object": {
            "id": "EFO_0007427",
            "name": "pericarditis"
          }
        },
        {
          "score": 0.7268268838011046,
          "object": {
            "id": "MONDO_0024643",
            "name": "myocardial disorder"
          }
        },
        {
          "score": 0.7262096005812022,
          "object": null
        }
      ]
    }
  }
}```

**Comment by d0choa** [comment]
More "internal server errors" happen with some targets:

```
query SimilarEntitiesQuery {
  target(ensemblId: "ENSG00000157764") {
    id
    approvedName
    similarEntities(threshold: 0.5, size: 15, entityNames: ["disease","drug","target"]) {
      score
      object {
        ... on Target {
          id
          approvedSymbol
          __typename
        }
      }
    }
  }
}
```

This might be due to the data not been loaded based on a clean ETL run. But we should monitor just in case.

**Comment by andrewhercules** [comment]
Ticket closed as similar issue resolved in #1781
